<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Ins√©rer votre texte ici&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="action_Ouvrir"/>
    <addaction name="action_Sauver"/>
   </widget>
   <widget class="QMenu" name="menu_Recherche">
    <property name="title">
     <string>&amp;Recherche</string>
    </property>
    <addaction name="action_Rechercher"/>
    <addaction name="action_Remplacer"/>
   </widget>
   <addaction name="menu_Fichier"/>
   <addaction name="menu_Recherche"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Ouvrir">
   <property name="text">
    <string>&amp;Ouvrir</string>
   </property>
  </action>
  <action name="action_Sauver">
   <property name="text">
    <string>&amp;Sauver</string>
   </property>
  </action>
  <action name="actionRechercher">
   <property name="text">
    <string>&amp;Rechercher</string>
   </property>
  </action>
  <action name="actionRemplacer">
   <property name="text">
    <string>&amp;Remplacer</string>
   </property>
  </action>
  <action name="action_Rechercher">
   <property name="text">
    <string>&amp;Rechercher</string>
   </property>
  </action>
  <action name="action_Remplacer">
   <property name="text">
    <string>&amp;Remplacer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
